src/examples/input-field/input-field-number.tsx

```diff
@@ -85,4 +91,7 @@ export class InputFieldNumberExample {
     private toggleRequired() {
         this.required = !this.required;
     }
+    private blurAway(){
+        this.el.blur()
```

Comment:
This only fixes the issue in this example, if I were to use the number input field in my own component, the problem would still be there. 
Also, this has the side effect of removing focus from the field when scrolling which is not a good user experience